applications/matter_bridge/src/bridge_shell.cpp

```diff
@@ -199,7 +199,15 @@ static void BluetoothScanResult(Nrf::BLEConnectivityManager::ScannedDevice *devi
 static int InsertBridgedDevicePincodeHandler(const struct shell *shell, size_t argc, char **argv)
 {
 	int bleDeviceIndex = strtoul(argv[0], NULL, 0);
-	unsigned int pincode = strtoul(argv[1], NULL, 0);
```

Comment:
If you only need to support decimal numbers, you can probably pass `10` instead of `0` as the last argument and that should fix the issue. I suppose the issue might have been that 0X numbers were parsed as octal, right?

Reply:
Oh right, I haven't thought it is that simple, thanks :) I haven't investigated it deeply, but the problem was that the numbers with leading 0s were not converted, as it would be decimal numbers.